public/app/features/dashboard/components/DashboardSettings/JsonEditorSettings.tsx

```diff
@@ -36,7 +39,7 @@ export function JsonEditorSettings({ dashboard, sectionNav }: SettingsPageProps)
         {dashboard.meta.canSave && (
           <div>
             <Button type="submit" onClick={onClick}>
-              Save changes
+              {t('dashboard-settings.json-editor-settings.save-button', 'Save changes')}
```

Comment:
since this is jsx, we can use the `<Trans>` component from `app/core/internationalization` instead